Package.swift

```diff
@@ -399,11 +399,16 @@ let package = Package(
             ]
         ),
 
+        .target(
+            /** Test for thread-santizer. */
+            name: "tsan_utils",
+            dependencies: []),
+
```

Comment:
Can we add this to `SPMTestSupport` instead of creating a new target?

Reply:
Sounds like a good plan. If that's all C code already.